test/unit/specs/FontStyle.spec.js

```diff
@@ -1,38 +1,42 @@
 import Vue from 'vue'
 import FontStyle from '@/components/FontStyle'
 
-// Test number of font colors showing
-describe('FontStyle.vue', () => {
-  it('Font Colors should be 11', () => {
-    const Constructor = Vue.extend(FontStyle)
-    const vm = new Constructor().$mount()
-    expect(vm.$data.fontColors.length).to.equal(11)
+describe('FontStyle.vue', function () {
+  let Constructor, vm
+
+  before(function () {
+    Constructor = Vue.extend(FontStyle)
+    vm = new Constructor().$mount()
   })
-})
 
-// Test number of font sizes showing
-describe('FontStyle.vue', () => {
-  it('Font Sizes should be 9', () => {
-    const Constructor = Vue.extend(FontStyle)
-    const vm = new Constructor().$mount()
-    expect(vm.$data.fontSizes.length).to.equal(9)
+  it('should have correct defaults', function () {
+    expect(vm.$props.fontColor).to.equal('')
+    expect(vm.$props.fontSize).to.equal('')
+    expect(vm.$props.fontWeight).to.equal('')
+    expect(vm.$props.fontStyle).to.equal('')
   })
-})
 
-// Test number of font styles showing
-describe('FontStyle.vue', () => {
-  it('Font Styles should be 3', () => {
-    const Constructor = Vue.extend(FontStyle)
-    const vm = new Constructor().$mount()
-    expect(vm.$data.fontStyles.length).to.equal(3)
+  it('should have correct colors', function () {
+    let expectedColors = ['Blue', 'Red', 'Green', 'Yellow', 'Purple', 'Pink', 'Teal', 'Lime', 'Orange', 'Brown', 'Grey']
```

Comment:
Can we hoist line 20, 26, 32, and 38 to the scope of the spec file iteself

Reply:
Done